ci/staging/deploy-app.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# Get terraform outputs
+pushd ../../infrasturcture/staging
+    REGION=$(terraform output region)
```

Comment:
We use [2 spaces for indention](https://github.com/newaperio/phoenix_starter/blob/67abc98a572a4b402a1ab2d83ee41969ed2bc8d7/.editorconfig#L3). This looks like it's 4, can you double check?